From: Yifeng <wuyifeng@nd.com.cn>
Date: Mon, 18 Oct 2021 07:01:00 +0000
Subject: AP9-2162: Set default page url to http://myPromethean.com.

The requirement is that it should load to http://myPromethean.com when the following cases:
* when Chromium opens.
* when the button of home is clicked.
* when a new tab is opened.

---
 components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java b/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java
--- a/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java
+++ b/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java
@@ -48,4 +48,4 @@
     public static final String NTP_HOST = "newtab";
-    public static final String NTP_URL = "chrome-native://newtab/";
-    public static final String NTP_NON_NATIVE_URL = "chrome://newtab/";
+    public static final String NTP_URL = "http://myPromethean.com/";
+    public static final String NTP_NON_NATIVE_URL = "http://myPromethean.com/";
     public static final String NTP_ABOUT_URL = "about:newtab";
--
2.17.1